articles/libraries/lock-ios/v2/migration.md

```diff
@@ -271,21 +270,8 @@ Auth0
 }
 ```
 
-#### Delegation
-
-Delegation is no available through Lock, but Lock v1 users who still need delegation can implement it via the the [Auth0.Swift library](/libraries/auth0-swift). The following is an example of delegation using Auth0.Swift:
+### Delegation
 
-```swift
-Auth0.authentication()
-  .delegation(withParameters: ["id_token" : "<AUTH0 ID TOKEN>", "api_type": "firebase"])
-  .start() { result in
-    switch result {
-    case .success(let payload):
-      // payload will have your firebase token
-    case .failure(let error):
-      // Handle Error
-    }
-  }
-```
+Delegation is not available through Lock. It can be implemented via a legacy method in [Auth0.Swift](/libraries/auth0-swift) for tenants which existed prior to June 2017, but delegation is deprecated and not recommended for most use cases. See the [Grant Types](/clients/client-grant-types#secure-alternatives-to-the-legacy-grant-types) documentation for more details.
```

Comment:
Maybe it's me, and the fact that I don't know the first thing about Auth0.Swift, but I followed the link and didn't understand what is the alternative that Auth0 recommends to me.